devex/vscode-extension/assets/baseSolutionTemplateFolder/[[project-name]].edl

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
If this is going into the master branch it should be:
// Copyright (c) Open Enclave SDK contributors.
Please remove all uses of copyright "Microsoft Corporation" for code intended to be in the main SDK repo.

Reply:
Fixed across all files.